### DISCUSSION TITLE
Recommended approach to using vue-apollo with vue-rx/RxJS?

### DISCUSSION THREAD

**Discussion by bramvanneerven** [Akryum/vue-apollo#268] [Q&A]
Hey @Akryum,

First of all, fantastic library. I really like GraphQL and being able to use it with Vue is great!
I have a question regarding using this library with `vue-rx`/`RxJS`. In `RxJS` we’d do something like this (obtained from: https://github.com/apollographql/apollo-client/issues/2788):

``` js
const Observable = require('rxjs').Observable;
const query = client.watchQuery({ query: MY_QUERY });
const query$ = Observable.from(query).map(...).subscribe(...);
```

Can you give some guidance on how to use this approach with `vue-apollo`?
I thought I read the `apollo-client` uses observables under the hood. They only provide an Angular example as part of the docs for implementing it though ([apollo docs](https://www.apollographql.com/docs/angular/basics/queries.html#rxjs)).
Is there a recommended approach to using `vue-apollo` together with `vue-rx`?
~~Also, the link referenced in `README.md` regarding the `watchQuery` API seems broken:~~

~~> You can use the apollo `watchQuery` options in the object, like:~~
~~>  - `fetchPolicy`~~
~~>  - `pollInterval`~~
~~>  - ...~~
~~>~~ 
~~> See the [apollo doc](https://www.apollographql.com/docs/react/reference/index.html#ApolloClient\.watchQuery) for more details.~~
Fixed! 🎉

Thanks in advance!

**Comment by bramvanneerven** [comment]
Adding @johnlindquist to this issue, as he might have some good ideas about this too!

**Comment by speedyankur** [comment]
Any fix for this one?
